.github/workflows/publish.yaml

```diff
@@ -9,8 +9,9 @@
 name: Upload Python Package
 
 on:
-  release:
-    types: [published]
+  push:
+    tags:        
+      - 'v*'           # Run on every git tag starting with "v". i.e: v1.5.0
```

Comment:
I think the tags here are just the version, no?

Reply:
Right, for example `git tag v0.0.5` + `git push` should trigger this workflow